apps/src/blockly/utils/toolbox/index.ts

```diff
@@ -0,0 +1,3 @@
+export {default as getToolboxDefinition} from './generateToolboxDefinition';
```

Comment:
@mikeharv let me know how you feel about this directory structure/organization. Happy to refine as necessary.